forward.c

```diff
@@ -519,6 +520,13 @@ int prepare_http_connect(int sd, struct auth_s *credentials, const char *thost)
 	data1->req = 1;
 	data1->method = strdup("CONNECT");
 	data1->url = strdup(thost);
+	data1->hostname = strdup(thost);
+	if ((pos = strchr(data1->hostname, ':')) != NULL) { // separate port
+		*pos = 0;
+		data1->port = atoi(pos + 1);
+	}
+	else
+		data1->port = 0;
```

Comment:
This else is redundant since `data1->port` is already equal to 0

Reply:
Yep, makes sense! I removed line 528 and 529!